### ISSUE TITLE
Minor (but important?) fixes

### ISSUE THREAD

**Issue by mintydev789** [thewca/wca-regulations#948]
12a2 "n may be also be omitted" should be changed to "n may also be omitted"
a2d1 What happens if there isn't a signature from the scrambler/scramble checker? Is that intentionally not stated because this has just been introduced? I can't quite tell if this really is mandatory or not.
a5a This means that if someone other than these people starts talking to a competitor they cannot do anything to ask them to stop other than ask the judge to do so. I recommend adding a point about allowing some involuntary remark that does not aid the competitor in any way at the discretion of the judge.
a6h it should say that noone except for a WCA delegate is allowed to even touch the puzzle in case of a dispute. I heard people say that, but it seems like it's not actually in the regs, you just can't align the puzzle.
a7b1 replace the / from "before/starting" with a space
a7c2 Just a question, is this mandatory? Cause I've never seen this enforced.
a7c4 I have seen multiple instances of delegates allowing a signature to be made after the fact, so the reg should probably be updated to include sportsmanship and what not.
b1b should it say that the blindfold must not be see through?
b4 should it instead say that the memo phase starts and a sight blocker is placed AFTER the first move is applied?
f3a but they can rotate the gears/clocks?
h1a3 shouldn't it be "arranged in A shape"?

**Comment by hanwu85** [comment]
Hi denthebro,

Thank you for the issues.  I will answer you as much as I can.
`12a2` Good catch!
`A2d1` Because there is a "must" in the sentence.  It is mandatory to do so.  However, at this moment, a missing signature is not punished, but the WCA Delegate should do their best to prevent this.  You can refer to the [Guideline](https://www.worldcubeassociation.org/regulations/guidelines.html#A2d1+) for more information.
`A5a` If someone is talking to the competitors during their solves, the judge should stop that.  Or the competitors can just ask for extra attempts.
`A6h` Actually, even the WCA Delegate should not touch the puzzle before it's resolved, so it is not written in the Regulations.
`A7b1` The before or starting is actually correct, because the inspection phase and the "starting" (hand position, touching puzzle...etc) of the solve are written in the same penalty area.  
`A7c2` It is mandatory, and it is enforced.  You must only sign it AFTER the solve.
`A7c4` It is allowed to sign after the attempt but it must be done before the end of the competition.  Please refer to [A7c4+](https://www.worldcubeassociation.org/regulations/guidelines.html#A7c4+)
`B1b` Please see [B4c1](https://www.worldcubeassociation.org/regulations/#B4c1) and [B1b+](https://www.worldcubeassociation.org/regulations/guidelines.html#B1b+)
`B4` I thought there was already a statement for that. We will check again.
`F3a` Of course not, rotating gear is applying moves and F3a exsited because pins are not considered in a solved state.
`H1a3` I am not a native speaker, so it might need others to clarify this one.

Thank you very much for these questions and suggestions.

**Comment by JonEsparaz** [comment]
`H1a3` Another good catch by @denthebro. It should be "arranged in _**a**_ shape" not "arranged in shape".

**Comment by mintydev789** [comment]
Sorry for some of the questions, because I found the answers to a few of them after I reread the guidelines. There are still some unresolved issues though.

A6h I still think it should implicitly say that nobody can touch the cube instead of adjust/rotate
A7b1 should still be fixed to something like "before the solve/while starting the solve"
F3a I was unable to find where it says that a gear/clock turn is considered a turn and thus will be penalized by a DNF

**Comment by hanwu85** [comment]
Don't be sorry, discussing is a good thing.  

`A6h` It already means no body can touch based on `A6g`, so I think it is fine at this moment.
`A7b1`  Personally, I think both are okay and the current one is cleaner.  If you strongly think the current one should be modified, you are welcome to add another issue just for this one.  Other team members can make decision.
`F3a`  If you read `12g3` , `A3c1`, then you will know moving a wheel/gear is considered making a move.

**Comment by mintydev789** [comment]
> Don't be sorry, discussing is a good thing.
> 
> `A6h` It already means no body can touch based on `A6g`, so I think it is fine at this moment.
> `A7b1` Personally, I think both are okay and the current one is cleaner. If you strongly think the current one should be modified, you are welcome to add another issue just for this one. Other team members can make decision.
> `F3a` If you read `12g3` , `A3c1`, then you will know moving a wheel/gear is considered making a move.

Oh, thank you, 12g3 is what I was missing.

**Comment by hanwu85** [comment]
For the `B4` question, I found the regulation it's [B4e](https://www.worldcubeassociation.org/regulations/#B4e)

**Comment by mintydev789** [comment]
> For the `B4` question, I found the regulation it's [B4e](https://www.worldcubeassociation.org/regulations/#B4e)

I am aware of this, but this means that the judge would have to quickly remove the sheet again and inexperienced judges would be even more confused. I think it would be simpler to tell them to put up the sheet after the first move is applied.

**Comment by hanwu85** [comment]
I think if we need to put it in, in would be better in the Guideline.  Could you suggest a guidline for this?

**Comment by mintydev789** [comment]
> I think if we need to put it in, in would be better in the Guideline. Could you suggest a guidline for this?

"ADDITION The judge should wait for the competitor to make their first move before putting up the sheet to avoid hindering their attempt if they choose to take off their blindfold and look at the puzzle again."
